aws-redshift-cluster/src/main/java/software/amazon/redshift/cluster/CreateHandler.java

```diff
@@ -1,62 +1,16 @@
-import software.amazon.awssdk.services.redshift.model.EnableLoggingRequest;
-import software.amazon.awssdk.services.redshift.model.EnableLoggingResponse;
-import software.amazon.awssdk.services.redshift.model.HsmClientCertificateNotFoundException;
-import software.amazon.awssdk.services.redshift.model.HsmConfigurationNotFoundException;
-import software.amazon.awssdk.services.redshift.model.InsufficientClusterCapacityException;
-import software.amazon.awssdk.services.redshift.model.InsufficientS3BucketPolicyException;
-import software.amazon.awssdk.services.redshift.model.InvalidClusterSnapshotStateException;
-import software.amazon.awssdk.services.redshift.model.InvalidClusterStateException;
-import software.amazon.awssdk.services.redshift.model.InvalidClusterSubnetGroupStateException;
-import software.amazon.awssdk.services.redshift.model.InvalidClusterTrackException;
-import software.amazon.awssdk.services.redshift.model.InvalidElasticIpException;
-import software.amazon.awssdk.services.redshift.model.InvalidRestoreException;
-import software.amazon.awssdk.services.redshift.model.InvalidRetentionPeriodException;
-import software.amazon.awssdk.services.redshift.model.InvalidS3BucketNameException;
-import software.amazon.awssdk.services.redshift.model.InvalidS3KeyPrefixException;
-import software.amazon.awssdk.services.redshift.model.InvalidSubnetException;
-import software.amazon.awssdk.services.redshift.model.InvalidTagException;
-import software.amazon.awssdk.services.redshift.model.InvalidVpcNetworkStateException;
-import software.amazon.awssdk.services.redshift.model.LimitExceededException;
-import software.amazon.awssdk.services.redshift.model.NumberOfNodesPerClusterLimitExceededException;
-import software.amazon.awssdk.services.redshift.model.NumberOfNodesQuotaExceededException;
-import software.amazon.awssdk.services.redshift.model.RedshiftException;
+import software.amazon.awssdk.services.redshift.model.*;
 import software.amazon.awssdk.services.redshift.model.ResourceNotFoundException;
-import software.amazon.awssdk.services.redshift.model.RestoreFromClusterSnapshotRequest;
-import software.amazon.awssdk.services.redshift.model.RestoreFromClusterSnapshotResponse;
-import software.amazon.awssdk.services.redshift.model.SnapshotScheduleNotFoundException;
-import software.amazon.awssdk.services.redshift.model.TagLimitExceededException;
-import software.amazon.awssdk.services.redshift.model.UnauthorizedOperationException;
-import software.amazon.cloudformation.exceptions.CfnAlreadyExistsException;
-import software.amazon.cloudformation.exceptions.CfnGeneralServiceException;
-import software.amazon.cloudformation.exceptions.CfnInvalidRequestException;
-import software.amazon.cloudformation.exceptions.CfnServiceLimitExceededException;
+import software.amazon.awssdk.services.redshift.model.UnsupportedOperationException;
+import software.amazon.cloudformation.exceptions.*;
```

Comment:
let's not do *, just keep the long long list of imported things :D best practice

Reply:
Done.